arches/app/models/models.py

```diff
@@ -1546,8 +1539,10 @@ class TileModel(models.Model):  # Tile
         blank=True,
         null=True,
         on_delete=models.CASCADE,
+        related_name="children",
+        related_query_name="child",
```

Comment:
Same decision point as above (`child_tile`, `child_tiles`?) Or `childtile` for symmetry with `parenttile`?

Reply:
Ah, the other one on NodeGroup already merged in #11614. But we can change it if we want to wordsmith a bit more.